static/app/views/issueDetails/streamline/foldSection.tsx

```diff
@@ -0,0 +1,170 @@
+  HYDRATION_DIFF = 'issue-details-hydration-diff',
+  REPLAY = 'issue-details-replay',
+
+  HPKP = 'issue-details-hpkp',
+  CSP = 'issue-details-csp',
+  EXPECTCT = 'issue-details-expectct',
+  TEMPLATE = 'issue-details-template',
+
+  BREADCRUMBS = 'issue-details-breadcrumbs',
+  DEBUGMETA = 'issue-details-debugmeta',
+  REQUEST = 'issue-details-request',
+
+  TAGS = 'issue-details-tags',
+  SCREENSHOT = 'issue-details-screenshot',
+
+  CONTEXTS = 'issue-details-contexts',
+  EXTRA = 'issue-details-extra',
+  PACKAGE = 'issue-details-package',
+  DEVICE = 'issue-details-device',
+  VIEW_HIERARCHY = 'issue-details-view-hierarchy',
+  ATTACHMENTS = 'issue-details-attachments',
+  SDK = 'issue-details-sdk',
+  GROUPING_INFO = 'issue-details-grouping-info',
+  RRWEB = 'issue-details-rrweb',
+}
+
+interface FoldSectionProps {
+  children: React.ReactNode;
+  /**
+   * Unique key to persist user preferences for initalizing the section to open/closed
+   */
+  sectionKey: FoldSectionKey;
+  /**
+   * Title of the section, always visible
+   */
+  title: React.ReactNode;
+  /**
+   * Actions associated with the section, only visible when open
+   */
+  actions?: React.ReactNode;
+  /**
+   * Should this section be initially open, gets overridden by user preferences
+   */
+  initialCollapse?: boolean;
+  /**
+   * Disable the ability for the user to collapse the section
+   */
+  preventCollapse?: boolean;
+}
+
+export function FoldSection({
+  children,
+  title,
+  actions,
+  sectionKey,
+  initialCollapse = false,
+  preventCollapse = false,
+  ...props
+}: FoldSectionProps) {
+  const [isCollapsed, setIsCollapsed] = useLocalStorageState(
+    `${LOCAL_STORAGE_PREFIX}${sectionKey}`,
+    initialCollapse
+  );
+  const [isLayerEnabled, setIsLayerEnabled] = useState(true);
+
+  const toggleCollapse = useCallback(
+    (e: React.MouseEvent) => {
+      e.preventDefault(); // Prevent browser summary/details behaviour
+      setIsCollapsed(collapsed => !collapsed);
+    },
+    [setIsCollapsed]
+  );
+
+  return (
+    <section {...props}>
+      <details open={!isCollapsed || preventCollapse}>
+        <Summary
+          preventCollapse={preventCollapse}
+          onClick={preventCollapse ? e => e.preventDefault() : toggleCollapse}
+        >
+          <InteractionStateLayer
+            hidden={preventCollapse ? preventCollapse : !isLayerEnabled}
+          />
+          <TitleWithActions>
+            {title}
+            {!preventCollapse && !isCollapsed && (
+              <div
+                onClick={e => e.stopPropagation()}
+                onMouseEnter={() => setIsLayerEnabled(false)}
+                onMouseLeave={() => setIsLayerEnabled(true)}
+              >
+                {actions}
+              </div>
+            )}
+          </TitleWithActions>
+          <IconWrapper preventCollapse={preventCollapse}>
+            <IconChevron direction={isCollapsed ? 'down' : 'up'} size="xs" />
```

Comment:
the chevrons on the stories page are pointing in the wrong direction

Reply:
Good catch!  Updated the storybook video, but it should be accurate now.